test-utils/runtime-tester/src/scenario_builder.rs

```diff
@@ -68,6 +73,21 @@ impl ScenarioBuilder {
         self
     }
 
+    /// Changes `home_dir` to the one specified as well as setting
+    /// `use_in_memory_store` to `false`.
+    ///
+    /// Note that if all you want is to use on-disk store you should call
+    /// `in_memory_store(false)`.  Setting home directory means that the caller
+    /// is responsible for cleaning the directory before and after the test is
+    /// run.  This setting is intended only for one-off cases where someone
+    /// wants to manually inspect the store and thus does not want the home
+    /// directory to be deleted after the test finishes.
+    pub fn home_dir(mut self, home_dir: impl Into<std::path::PathBuf>) -> Self {
+        self.scenario.use_in_memory_store = false;
+        self.scenario.home_dir = Some(home_dir.into());
+        self
+    }
```

Comment:
An alternative would be to add just `fn keep_temp_dir(self, yes: bool)` and print the path at the end of the test. I usually do that approach, as that way I don't have to think about where to put it, and just let the computer decide

https://docs.rs/tempfile/3.2.0/tempfile/struct.TempDir.html#method.into_path